.github/workflows/build.yml

```diff
@@ -57,7 +57,7 @@ jobs:
   # Build both Uno.UI/WinUI2/UWP and Uno.WinUI/WinUI3/WindowsAppSDK versions of our packages using a matrix
   build:
     needs: [Xaml-Style-Check]
-    runs-on: windows-latest-large
+    runs-on: windows-2022
```

Comment:
For the build we probably want to look at the large one here for at least build times. Though 45min on the default VM isn't bad, wonder if they've made other optimizations to virtual storage or something too...?

Reply:
Agree. We'll come back to this if/when it becomes necessary.